en/components/splitter.md

```diff
@@ -150,6 +150,45 @@ Keyboard navigation is available by default in the splitter component. When you
 - `Ctrl + Arrow Left` - Expands/Collapses a pane in a horizontal splitter
 - `Ctrl + Arrow Right` - Expands/Collapses a pane in a horizontal splitter
 
+### Styling
+You can change the default styles of the **igxSplitter** component by creating a new theme for it.
+
+#### Demo
+This is the final result from applying your new theme.
+
+<div class="sample-container loading" style="height: 340px">
+    <iframe id="splitter-styling-sample-iframe" data-src='{environment:demosBaseUrl}/layouts/splitter-styling-sample' width="100%" height="100%" seamless="" frameBorder="0" class="lazyload"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="splitter-styling-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+#### Creating a component theme
+First you need to [create a custom palette](../themes/palette.md).
```

Comment:
This is not strictly true. Not to mention you never pass the palette to the theme anyways.

Reply:
I changed the styling section to be similar to the Avatar's one. Let me know if additional changes are needed.